src/Octoshift/Commands/CommandBase.cs

```diff
@@ -0,0 +1,11 @@
+using System.CommandLine;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace OctoshiftCLI.Commands;
+
+public abstract class CommandBase<TArgs, THandler> : Command where TArgs : class where THandler : class
```

Comment:
```suggestion
public abstract class CommandBase<TArgs, THandler> : Command where TArgs : class where THandler : ICommandHandler<TArgs>
```

Reply:
Great catch! I totally forgot to mark it as `ICommandHander` :)